test/extensions/filters/common/rbac/matchers_test.cc

```diff
@@ -227,28 +227,50 @@ TEST(IPMatcher, IPMatcher) {
   downstream_remote_cidr.set_address_prefix("8.9.10.11");
   downstream_remote_cidr.mutable_prefix_len()->set_value(32);
 
-  checkMatcher(IPMatcher(connection_remote_cidr, IPMatcher::Type::ConnectionRemote), true, conn,
-               headers, info);
-  checkMatcher(IPMatcher(downstream_local_cidr, IPMatcher::Type::DownstreamLocal), true, conn,
-               headers, info);
-  checkMatcher(IPMatcher(downstream_direct_remote_cidr, IPMatcher::Type::DownstreamDirectRemote),
-               true, conn, headers, info);
-  checkMatcher(IPMatcher(downstream_remote_cidr, IPMatcher::Type::DownstreamRemote), true, conn,
-               headers, info);
+  auto connection_remote_matcher =
+      IPMatcher::create(connection_remote_cidr, IPMatcher::Type::ConnectionRemote);
+  ASSERT_TRUE(connection_remote_matcher.ok());
```

Comment:
`ASSERT_OK(connection_remote_matcher);` from status_utility.h

(That way if it fails the failure message tells you the status, vs. this way if it fails all you get is 'false')

(and throughout)

Reply:
Thanks, I'll replace it with `ASSERT_OK(...)` as suggested.